app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html

```diff
@@ -23,7 +23,7 @@
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
 
 @this(
-        main_template: main_template
+        main_template_old: main_template_old
```

Comment:
no need to change  main_template to main_template_old here keep it `main_template:main_template_old`

Reply:
Fixed, not sure when that page changed

Reply:
I can still see this as main_template_old: main_template_old, have the changes been pushed?